### DISCUSSION TITLE
期望模拟测试的题库能更新下版本，或者和新版题库同步更新

### DISCUSSION THREAD

**Discussion by ghost** [MlgmXyysd/Xiaomi-BootLoader-Questionnaire#47] [Ideas]
现在的模拟测试题库还是旧版的，本来想着靠背熟选答案的，结果新题目不在模拟测试内。
![2](https://github.com/MlgmXyysd/Xiaomi-BootLoader-Questionnaire/assets/77822042/262d3f5f-7b58-458b-888b-2468f8a15d7d)

**Comment by MlgmXyysd** [comment]
@cjybyjk
